Ideal-Media-Solutions-LLC/chadslist
# Basic express server set up

## Story / Task

Here goes the link and info of the task/story.  
Example:

see [Backend express server set up](https://trello.com/c/hiSBV2oN)


## What this PR does?

This sets up a basic express server that listens on port 3001 and return "Hello World!"


### Before 

We didn't have a backend server.

### After

We have a backend server that doesn't do anything useful but it is a start!

### How to test / reproduce

- Make sure you `npm install` to install new dependencies
- 'npm run backend` will run the backend server in watch mode using nodemon
- `http//localhost:3000` should display `Hello World!`

## Mentions

@DevonL1010 @vinhle000 @davidhannn @Doomslair @stevenxm-chen @yxxy666 @ajpsyk 

Thanks for looking over this!
